unit_07/hypothesis_test_practice_activity.Rmd

```diff
@@ -1,24 +1,41 @@
 ---
-title: "hypothesis_test_practice_activity"
+title: "Hypothesis Test Practice Activity"
 author: "Alex Hughes"
 date: "10/4/2021"
-output: html_document
+output: bookdown::gitbook
 ---
 
+# Hypothesis Test Practice Activity
+
+In this short activity, you're going to write, and execute a short series of hypothesis tests using the `R` estimating framework. 
```

Comment:
not sure what the R estimating framework is.  just use R?  I don't think short is accurate at the moment :D 
```suggestion
In this activity, you're going to execute a series of hypothesis tests using R. 
```